test/helpers/reset-config.ts

```diff
@@ -4,7 +4,7 @@ export default async () => {
   const files = await readdir('.');
 
   await Promise.all(files
-    .filter((file) => /\.griffin-config\..+\.json/.test(file))
+    .filter((file) => /\.griffin-config\..+\.yaml/.test(file))
```

Comment:
What sort of errors?  Did you happen to record any?

Reply:
you will get strange `Exit Code 1` errors in the integration tests if the `.griffin.<env>.whatever` files are not removed between each test run

I did not record the errors while debugging, but you can reliably simulate it by replacing line `7` with "bad" regex, i.e.

```js
    .filter((file) => /\.griffin-config\..+\.txt/.test(file))
```

**don't remove the `.filter(...)` statement**.  If you do this, it will recursively remove the `griffin-cli` files from your hard drive.  Ask me how I know 😅 .

Reply:
I've been considering moving the files generated into a `.tmp` directory and deleting the whole thing.  Sounds like that would fix this issue, too.